pennylane/qchem/observable_hf.py

```diff
@@ -122,41 +122,47 @@ def qubit_observable(o_ferm, cutoff=1.0e-12):
     return o_qubit
 
 
-def jordan_wigner(op):
+def jordan_wigner(op, notation="physicist"):
     r"""Convert a fermionic operator to a qubit operator using the Jordan-Wigner mapping.
 
     For instance, the one-body fermionic operator :math:`a_2^\dagger a_0` should be constructed as
-    [2, 0] and the two-body operator :math:`a_4^\dagger a_3^\dagger a_2 a_1` should be constructed
-    as [4, 3, 2, 1].
+    [2, 0]. The two-body operator :math:`a_4^\dagger a_3^\dagger a_2 a_1` should be constructed
+    as [4, 3, 2, 1] with ``notation='physicist'``. If ``notation`` is set to ``'chemist'``, the
+    two-body operator [4, 3, 2, 1] is constructed as :math:`a_4^\dagger a_3 a_2^\dagger a_1`.
 
     Args:
         op (list[int]): the fermionic operator
+        notation (str): notation specifying the order of the two-body fermionic operators
 
-    Returns
-        tuple(list[complex], list[list[int, str]]): list of coefficients and the qubit-operator terms
+    Returns:
+        tuple(list[complex], list[Operation]): list of coefficients and qubit operators
```

Comment:
This is actually inconsistent with the function as the function some times returns 0. I would recommend in the cases where it returns 0 to instead return ([0], [Identity(wires)]). 

This would also clean up the tests so that you don't have the extra if clause.

Reply:
Corrected.